slack-api-client/src/main/java/com/slack/api/SlackConfig.java

```diff
@@ -23,9 +23,21 @@
 
 /**
  * The basic configuration of this SDK. Some settings can be propagated to submodules such as Bolt.
+ * <p>
+ * Please note that, if you are fine with the same settings for all Slack instances across an application,
+ * using a singleton instance of this config class is highly recommended.
+ * Also, a Slack instance is thread-safe, so you can use singleton for it too.
+ * <p>
+ * If you create a new SlackConfig instance for each Slack instance creation, each SlackConfig object
+ * can create thread pools for maintaining metrics data and async executions.
+ * In most use cases, this should not be intended. To avoid this, consider reusing SlackConfig objects
+ * as much as possible.
+ * <p>
+ * An alternative way is to set the statsEnabled flag to false.
```

Comment:
I just realized that these docs are hosted on sonatype.org, and as far as I can tell, can't be previewed locally in Jekyll 😢 .
I wanted to check to see if the references to various classes (i.e. `SlackConfig`) and maybe even properties (i.e. `statsEnabled`) would link properly. I suppose we won't be able to tell until we merge + deploy?

Reply:
FYI: You can package javadoc.jar files by running mvn command and can unzip the file to check the generated HTML!